files/knative-external-ingressroute-gateway.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: contour-external
```

Comment:
Is this required in external mode?

Reply:
Good catch! No, its not needed. Will fix